VDF.GUI/Views/MainWindow.xaml

```diff
@@ -1295,6 +1295,12 @@
                                                         IsChecked="{Binding Source={x:Static Settings:SettingsFile.Instance}, Path=GeneratePreviewThumbnails}"
                                                         IsThreeState="False"
                                                         ToolTip.Tip="Deactivate to save memory by disabling preview images of duplicates" />
+                                                    <CheckBox
+                                                        VerticalAlignment="Center"
+                                                        Content="Exclude Hard Links"
```

Comment:
Change it to "Exclude Hard Links (Windows Only)"

Reply:
> Not yet but I think I can do as most of the regressions in .NET 7

Where you able to compile your changes using .NET 6?

> Change it to "Exclude Hard Links (Windows Only)". Or leave a note in the tooltip that it only works on Windows.

I'll see if I can hide this option when not running on Windows.

Reply:
Not hiding, just rename. So people know this feature exists. The code above won't compile as `LibraryImport` is a .NET 7 feature. Like I said you can change it back to `DllImport`.